modules/shortcodes/js/blocks/vr-block.jsx

```diff
@@ -0,0 +1,94 @@
+/* global wp */
+/* eslint react/react-in-jsx-scope: 0 */
+
+( function( blocks, components, i18n ) {
+	const {
+		registerBlockType,
+		UrlInput
+	} = blocks;
+	const {
+		Placeholder,
+		SelectControl
+	} = components;
+	const { __ } = i18n;
+
+	registerBlockType( 'gutenpack/vr', {
+		title: __( 'VR Image' ),
+		icon: 'embed-photo',
+		category: 'embed',
+		supportHTML: false,
+		attributes: {
+			url: {
+				type: 'string',
+			},
+			view: {
+				type: 'string',
+			}
+		},
+
+		edit: props => {
+			const attributes = props.attributes;
+
+			const onSetUrl = value => props.setAttributes( { url: value } );
+			const onSetView = value => props.setAttributes( { view: value } );
+
+			const renderEdit = () => {
+				if ( attributes.url && attributes.view ) {
+					return (
+						<div className={ props.className }>
+							<iframe
+								title={ __( 'VR Image' ) }
+								allowFullScreen="true"
+								frameBorder="0"
+								width="100%"
+								height="300"
+								src={ 'https://vr.me.sh/view/?view=' + encodeURIComponent( attributes.view ) + '&url=' + encodeURIComponent( attributes.url ) }
+							/>
+						</div>
+					);
+				}
+				return (
+					<div>
+						<Placeholder
+							key="placeholder"
+							instructions={ __( 'Enter URL to VR image' ) }
```

Comment:
In the latest version of `i18n`, it follows PHP and `__` takes a domain name as 2nd param. See: https://github.com/WordPress/packages/tree/master/packages/i18n#api

> `__( text: string, domain: string ): string`
> 
> Retrieve the translation of text.
> 
> See: https://developer.wordpress.org/reference/functions/__/

Reply:
Fixed, thanks